Remove validation step for HTTPS JWT URLs, allowing HTTP URLs as well (#418)
kyma-project/api-gateway: docs/jwt-ory-to-istio.md

  </details>
</div>

> >**CAUTION:** Both `jwks_urls` and `trusted_issuers` must be valid URLs, please consider using HTTPS endpoints only, however HTTP is also allowed.

Reviewer: ```suggestion
>**CAUTION:** Both `jwks_urls` and `trusted_issuers` must be valid URLs. Although HTTP is allowed, it is recommended that you use only HTTPS endpoints. 
```
